generators/app/index.js

```diff
@@ -522,52 +509,15 @@ module.exports = class JHipsterAppGenerator extends BaseBlueprintGenerator {
       gitCommit() {
         if (!this.options.skipGit && this.isGitInstalled()) {
-          if (this.gitInitialized) {
-            this.debug('Committing files to git');
-            const done = this.async();
-            this.gitExec('log --oneline -n 1 -- .', { trace: false }, (code, commits) => {
-              if (code !== 0 || !commits || !commits.trim()) {
-                // if no files in Git from current folder then we assume that this is initial application generation
-                this.gitExec('add .', { trace: false }, code => {
-                  if (code === 0) {
-                    let commitMsg = `Initial version of ${this.jhipsterConfig.baseName} generated by JHipster-${this.jhipsterConfig.jhipsterVersion}`;
-                    if (this.jhipsterConfig.blueprints && this.jhipsterConfig.blueprints.length > 0) {
-                      const bpInfo = this.jhipsterConfig.blueprints
-                        .map(bp => `${bp.name.replace('generator-jhipster-', '')}-${bp.version}`)
-                        .join(', ');
-                      commitMsg += ` with blueprints: ${bpInfo}`;
-                    }
-                    this.gitExec(`commit -m "${commitMsg}" -- .`, { trace: false }, code => {
-                      if (code === 0) {
-                        this.log(chalk.green.bold(`Application successfully committed to Git from ${process.cwd()}.`));
-                      } else {
-                        this.log(chalk.red.bold(`Application commit to Git failed from ${process.cwd()}. Try to commit manually.`));
-                      }
-                      done();
-                    });
-                  } else {
-                    this.warning(
-                      `The generated application could not be committed to Git, because ${chalk.bold('git add')} command failed.`
-                    );
-                    done();
-                  }
-                });
-              } else {
-                // if found files in Git from current folder then we assume that this is application regeneration
-                // if there are changes in current folder then inform user about manual commit needed
-                this.gitExec('diff --name-only .', { trace: false }, (code, diffs) => {
-                  if (code === 0 && diffs && diffs.trim()) {
-                    this.log(
-                      `Found commits in Git from ${process.cwd()}. So we assume this is application regeneration. Therefore automatic Git commit is not done. You can do Git commit manually.`
-                    );
-                  }
-                  done();
-                });
-              }
-            });
-          } else {
-            this.warning('The generated application could not be committed to Git, as a Git repository could not be initialized.');
+          let commitMsg = `Initial version of ${this.jhipsterConfig.baseName} generated by JHipster-${this.jhipsterConfig.jhipsterVersion}`;
```

Comment:
Not related to this PR, but I think our initial commit is not nice. We should include the package name from official NPM repo, something like generator-jhipster@7.0.1 so it can be easy to copy/paste etc
What do you think ?

Reply:
I agree.